cmd/meroxa/root/apps/describe.go

```diff
@@ -75,6 +76,8 @@ func (d *Describe) Execute(ctx context.Context) error {
 	d.logger.Info(ctx, output)
 	d.logger.JSON(ctx, app)
 
+	dashboardURL := fmt.Sprintf("https://dashboard.meroxa.io/apps/%s/detail", app.Name)
```

Comment:
Not a review, just a comment. We have the `Name` to improve user usability, but should we use that as the default value for our URLs? The reasoning is that using UUID guarantees that the resource is unique and the same 100% of the time, and names, especially in a bigger organization, can generate start to generate conflicts/confusion.

Reply:
names are unique within the account, and you can't see the dashboard without being logged in to the account. we would have much bigger problems if a customer wanted us to change our name schema than this URL. 

    "application_name_account_id" UNIQUE, btree (name, account_id) WHERE deleted_at IS NULL
    
are you describing the situation in which a user belongs to multiple accounts and each account has an app with the same name?  that's something that the customer has control of.  i'd hope they can come up with a naming convention that doesn't confuse themselves.